app/assets/javascripts/services/bridge.ts

```diff
@@ -1,24 +1,30 @@
 /** Platform-specific (i-e Electron/browser) behavior is handled by a Bridge object. */
 export interface Bridge {
-  getKeychainValue(): Promise<unknown>;
+  getKeychainValue(): Promise<any>;
```

Comment:
Why the switch to `any`? `unknown` is correct here because we're fetching something from a third-party resource (OS keychain, localstorage). We should treat that result with caution, and at worst it's just a cast to `string` away from being usable.